src/platform/errors/index.ts

```diff
@@ -21,7 +21,8 @@ export function populateTelemetryWithErrorInfo(props: Partial<TelemetryErrorProp
         // Helps us determine that we are rejecting with errors in some places, in which case we aren't getting meaningful errors/data.
         props.failureSubCategory = 'errorisstring';
     }
-    const stdErr = (error instanceof BaseError ? error.stdErr : error.stack) || '';
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const stdErr = (error as any).stdErr ? (error as any).stdErr : error.stack || '';
```

Comment:
```suggestion
    const stdErr = (error as BaseError ).stdErr ? (error as BaseError ).stdErr : error.stack || '';
```

Reply:
I think its better to cast it to an expected type, else tomorrow if we rename `stdErr` we don't benefit from types to let us know where we forgot.